x-pack/plugins/ml/server/routes/schemas/alerting_schema.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+
+export const mlAnomalyDetectionAlertParams = schema.object({
+  jobSelection: schema.object(
+    {
+      jobIds: schema.maybe(schema.arrayOf(schema.string())),
+      groupIds: schema.maybe(schema.arrayOf(schema.string())),
+    },
+    {
+      validate: (v) => {
+        if (!v.jobIds?.length && !v.groupIds?.length) {
+          return 'List of job ids or group ids is required';
```

Comment:
```suggestion
          return 'List of job IDs or group IDs is required';
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/89286/commits/4acf5401e6441e83ceb4cc237dda2a1b46fe91b2